lib/pure/options.nim

```diff
@@ -39,6 +39,30 @@ runnableExamples:
 ## inherits from `system.Defect` and should therefore never be caught.
 ## Instead, rely on checking if the option contains a value with the
 ## `isSome <#isSome,Option[T]>`_ and `isNone <#isNone,Option[T]>`_ procs.
+##
+##
+## Pattern matching
+## ================
+##
+## .. note:: This requires the [fusion](https://github.com/nim-lang/fusion) package.
+##
+## [fusion/matching](https://nim-lang.github.io/fusion/src/fusion/matching.html)
+## supports pattern matching on `Option`s, with the `Some(<pattern>)` and
+## `None()` patterns.
+##
+## .. code-block:: nim
```

Comment:
can you at least use `##[` block comments so that the code is more easily copy-pastable?